CRM/Contact/BAO/Query.php

```diff
@@ -5089,44 +5089,27 @@ public function setSkipPermission($val) {
    */
   public function summaryContribution($context = NULL) {
     list($innerselect, $from, $where, $having) = $this->query(TRUE);
+    if ($this->_permissionWhereClause) {
+      $where .= " AND " . $this->_permissionWhereClause;
+    }
+    if ($context == 'search') {
+      $where .= " AND contact_a.is_deleted = 0 ";
+    }
 
     // hack $select
     $select = "
 SELECT COUNT( conts.total_amount ) as total_count,
        SUM(   conts.total_amount ) as total_amount,
        AVG(   conts.total_amount ) as total_avg,
        conts.currency              as currency";
-    if ($this->_permissionWhereClause) {
-      $where .= " AND " . $this->_permissionWhereClause;
-    }
-    if ($context == 'search') {
-      $where .= " AND contact_a.is_deleted = 0 ";
-    }
 
-    $query = $this->appendFinancialTypeWhereAndFromToQueryStrings($where, $from);
+    $this->appendFinancialTypeWhereAndFromToQueryStrings($where, $from);
 
-    // make sure contribution is completed - CRM-4989
-    $completedWhere = $where . " AND civicrm_contribution.contribution_status_id = 1 ";
```

Comment:
I've been moving this clause into the relevant queries - there just isn't enough gained by sharing bits of queries & it makes it more complicated